package.json

```diff
@@ -40,7 +40,7 @@
         "@types/react-dom": "17.0.5",
         "@typescript-eslint/eslint-plugin": "5.1.0",
         "@typescript-eslint/parser": "5.1.0",
-        "copy-webpack-plugin": "8.1.1",
```

Comment:
considering the number of new packages in package-lock.json, and the diff between the [two versions](https://github.com/webpack-contrib/copy-webpack-plugin/compare/v8.1.1...v12.0.2), this somehow innocuous change is hard to review.

this is one reason for cloudflare/pp-browser-extension fork to move away from webpack